vars/deployKubernetesSnapshot.groovy

```diff
@@ -70,7 +70,9 @@ def call(body) {
 
         if (extraYAML){
             writeFile file: "./extra.yml", text: extraYAML
-            sh "kubectl apply -n ${namespace} -f ./extra.yml"
+            retry |3| {
```

Comment:
not sure if pipes work - I think we tend to use brackets, might be best to keep the same?  
i.e. 
```
retry (3) {
...
}
```